ols/app/endpoints/feedback.py

```diff
@@ -64,7 +65,10 @@ def store_feedback(user_id: str, feedback: dict) -> None:
         logger.debug(f"creating feedback storage directories '{storage_path}'")
         storage_path.mkdir(parents=True)
 
-    data_to_store = {"user_id": user_id, **feedback}
+    
+    timestamp = datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S')
+
+    data_to_store = {"user_id": user_id,"timestamp": timestamp, **feedback}
```

Comment:
it looks ok, but linter will probably tell you that space is missing after comma